src/map_main.cpp

```diff
@@ -345,53 +348,94 @@ int pandora_map(int argc, char *argv[]) {
 
     std::cout << now() << "Find PRG paths and write to files:" << std::endl;
 
+
+
+    //paralell region!
+    //shared variable - synced with critical(consensus_fq)
     Fastaq consensus_fq(true, true);
-    VCF master_vcf;
 
-    VCFRefs vcf_refs;
-    std::string vcf_ref;
-    std::vector<KmerNodePtr> kmp;
-    std::vector<LocalNodePtr> lmp;
+    //shared variable - synced with critical(master_vcf)
+    VCF master_vcf;
 
+    //shared variable - synced with critical(candidate_regions)
     CandidateRegions candidate_regions;
 
+    //shared variable - will denote which nodes we have to remove after the parallel loop
+    //synced with critical(nodesToRemove)
+    std::vector<pangenome::NodePtr> nodesToRemove;
```

Comment:
Please change to snake_case

Reply:
Done in https://github.com/leoisl/pandora/commit/5b329a78d89eaa0f85592258a8fb378677f71587